<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="dataSourceStorageLocal" created-in="PY-242.24807.21">
    <data-source name="@localhost" uuid="d664cf5e-434c-401c-b9fa-54e20face47d">
      <database-info product="" version="" jdbc-version="" driver-name="" driver-version="" dbms="MYSQL" />
      <secret-storage>master_key</secret-storage>
      <schema-mapping />
    </data-source>
    <data-source name="Django default" uuid="f8d29175-607f-455d-9d7b-69d50d276168">
      <database-info product="MySQL" version="5.5.5-10.4.32-MariaDB" jdbc-version="4.2" driver-name="MySQL Connector/J" driver-version="mysql-connector-j-8.2.0 (Revision: 06a1f724497fd81c6a659131fda822c9e5085b6c)" dbms="MARIADB" exact-version="10.4.32" exact-driver-version="8.2">
        <extra-name-characters>#@</extra-name-characters>
        <identifier-quote-string>`</identifier-quote-string>
      </database-info>
      <case-sensitivity plain-identifiers="exact" quoted-identifiers="exact" />
      <schema-mapping />
    </data-source>
  </component>
</project>